src/modules/Experience/models/HeadModel.tsx

```diff
@@ -0,0 +1,47 @@
+import { useGLTF } from "@react-three/drei"
+import { PointsLoader } from "../components"
+import { GroupProps } from "@react-three/fiber"
+import { Group, Object3DEventMap, Texture } from "three"
+import { Color } from '@react-three/fiber';
+import { useEffect, useMemo, useState } from "react";
+
+interface HeadModelProps extends GroupProps {
+  selectedColor: Color,
+  map: Texture
+}
+
+const HeadModel = ({ selectedColor, map, ...props }: HeadModelProps) => {
+  const [gltfScene, setGltfScene] = useState<Group<Object3DEventMap> | null>(null);
+  const { scene } = useGLTF('/models/head-3.glb')
+  /**
+   * Model created by @Hicham2012 and
+   * inspired from this tutorial ↓
+   * https://youtu.be/AlPPYkZg9D4?si=L_chsWglPCB2DAGk
+   */
+  // Let's ignore the issue for now
+  // @ts-ignore
+  const head = scene.children[0].geometry.clone()
+
+  useEffect(() => {
+    setGltfScene(scene)
+  }, [])
+  const cachedModel = useMemo(() => {
+    if (gltfScene) {
+      return (
+        <group
```

Comment:
```suggestion
    if (gltfScene) {
    const head = gltfScene.children[0].geometry.clone()
      return (
        <group
```

You forgot to use the  gltfScene... head still use the old scene.

Reply:
@Hicham2012  here